group.py

```diff
@@ -2,4 +2,37 @@
 
 # Your code to go here...
 
-my_group =
+my_group = {
+    'Jill' : {
+        'relationship': {
+            'Zalika': 'friend',
+            'John': 'partner'
+        },
+        'age': 26,
+        'job': 'Biologist'
+    },
+    'Zalika' : {
+        'relationship': {
+            'Jill': 'friend'
+        },
+        'age': 28,
+        'job': 'Artist'
+    },
+    'John' : {
+        'relationship': {
+            'Jill': 'partner'
+        },
+        'age': 27,
+        'job': 'Writer'
```

Comment:
Looks good, but you could also add a reciprocal tag to the relationships, e.g.

`'relationship': {
            'Jill': 'partner',
            'Reciprocal ': True       
        },`